Content.Server/Zombies/ZombieSystem.Transform.cs

```diff
@@ -262,6 +263,11 @@ public void ZombifyEntity(EntityUid target, MobStateComponent? mobState = null)
                 RemComp(target, handsComp);
             }
 
+            if (TryComp<SharedPullerComponent>(target, out var pullerComp))
+            {
+                RemComp(target, pullerComp);
+            }
```

Comment:
```suggestion
            RemComp<SharedPullerComponent>(target);
```

remcomp doesn't error with the generic overload if the component isn't present, btw

Reply:
Applied